src/content/en/2020/pwa.md

```diff
@@ -22,3 +22,257 @@ queries: 14_PWA
 #featured_stat_label_3: TODO
 unedited: true
 ---
+
+PWA
```

Comment:
Do we need this?

Reply:
Nope, the H1 and hero image are added automatically by the template. See [2019/pwa.md](https://raw.githubusercontent.com/HTTPArchive/almanac.httparchive.org/main/src/content/en/2019/pwa.md) for example.